src/components/Videos.vue

```diff
@@ -84,6 +84,8 @@ export default {
 		options() {
 			return {
 				autoplay: this.active === true,
+				// Used to reset the video streams https://github.com/sampotts/plyr#javascript-1
+				blankVideo: '../img/blank.mp4',
```

Comment:
do that even works?
Shouldn't it be a geenrateImgPath or something? :thinking:

Reply:
Yes